samples/Samples.HttpMessageHandler/HttpClientHelpers.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace;
+
+namespace Samples.HttpMessageHandler
+{
+    internal static class HttpClientHelpers
```

Comment:
This file looks like a duplicate of `samples/Samples.HttpClientDriver/HttpClientHelpers.cs`. Can we move this to one of the projects shared by tests/samples?

Reply:
It shouldn't be in there. I removed from the project but didn't delete the file yet.